lecture/forms.py

```
@@ -0,0 +1,20 @@
+
+class LectureForm(forms.ModelForm):
+    class Meta:
+        model = Lecture
+        fields = ['title', 'classroom']
+
+class LecturePDFForm(forms.ModelForm):
+    class Meta:
+        model = LecturePDF
+        fields = ['pdf_file','lecture']
+
+    def save(self, commit=True, lecture=None):
```

Ruff (0.8.2)

Missing return type annotation for public function `save`

(ANN201)